lib/senkyoshi.rb

```diff
@@ -55,6 +56,9 @@ def self.create_canvas_course(resources, zip_name)
     course = CanvasCc::CanvasCC::Models::Course.new
     course.course_code = zip_name
     resources.each do |resource|
+      # Skips resources that are files.
```

Comment:
Yes it should, it was changed but not pushed, we are getting a new version to review now